### PR TITLE
Add quick info at location

### PR DESCRIPTION
**Pull request by atscott**
Support for template that aren't inline depends on #39002. Several microsyntax tests are currently failing and depend on #39036 and follow-up fix to the hybrid visitor to return the correct attribute binding.

### CODE REVIEW

**Comment by kyliau** [comment]
Could you please separate the PR into compiler and language service changes?
Language service does not require g3 presubmit, separating them not only makes reviewing easier, but also makes changes easier to land.

File: packages/compiler/src/i18n/extractor_merger.ts [hunk 1]
```diff
@@ -394,10 +394,12 @@ class _Visitor implements html.Visitor {
         const nodes = this._translations.get(message);
         if (nodes) {
           if (nodes.length == 0) {
-            translatedAttributes.push(new html.Attribute(attr.name, '', attr.sourceSpan));
+            translatedAttributes.push(
+                new html.Attribute(attr.name, '', attr.sourceSpan, null, undefined, undefined));
```
Comment (AndrewKushnir) [inline packages/compiler/src/i18n/extractor_merger.ts:398]: nit: there are few places in compiler code where we add inline comments for arguments where it's not immediately obvious what this argument is. May be we can use this approach here (and in some places below) too?
```suggestion
                new html.Attribute(attr.name, '', attr.sourceSpan, null /* keySpan */, undefined /* valueSpan */, undefined /* i18n */));
```

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: fw-i18n

**Comment by atscott** [comment]
Closing, will re-open once the required compiler PRs are merged

**Pull request closed without merging**